python/pyarrow/tests/test_array.py

```diff
@@ -2469,16 +2469,30 @@ def test_buffers_nested():
     assert struct.unpack('4xh', values) == (43,)
 
 
-def test_nbytes_sizeof():
+def test_total_buffer_size():
     a = pa.array(np.array([4, 5, 6], dtype='int64'))
-    assert a.nbytes == 8 * 3
+    assert a.get_total_buffer_size() == 8 * 3
```

Comment:
Can you keep both `assert a.nbytes == ..` and `assert a.get_total_buffer_size() == ..` here (and same for the ones below)